suspendapp-ktor/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+@Suppress("DSL_SCOPE_VIOLATION")
+plugins {
+  id(libs.plugins.kotlin.jvm.get().pluginId)
+
+  alias(libs.plugins.arrowGradleConfig.formatter)
+  alias(libs.plugins.arrowGradleConfig.publish)
+  alias(libs.plugins.arrowGradleConfig.versioning)
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies{
+  implementation(libs.coroutines)
+  implementation(libs.arrow.fx)
+  implementation(libs.ktor.core)
+  implementation(libs.ktor.netty)
```

Comment:
We only need `core` here without specifying a _runtime_ `ApplicationEngine`. That way we can publish this library for _all_ targets that Ktor Core also provides.